docs/api/resource-reaper.md

```diff
@@ -0,0 +1,29 @@
+# Resource Reaper
+
+Testcontainers automatically assigns a Resource Reaper session id to each Docker resource. After the tests are finished — whether they are successful or not — [Ryuk][moby-ryuk] will take care of the remaining Docker resources and removes them. You can change the Resource Reaper session and group Docker resources together. Right now, only Linux containers are supported.
+
+!!!tip
+
+    Whenever possible, do **not** disable the Resource Reaper. It keeps your machine and CI/CD environment clean.
```

Comment:
In Go we use this message: https://golang.testcontainers.org/features/garbage_collector/#ryuk from https://github.com/testcontainers/testcontainers-go/blame/main/docs/features/garbage_collector.md#L40

<img width="784" alt="Screenshot 2022-10-26 at 10 28 25" src="https://user-images.githubusercontent.com/951580/197975071-3e440182-81ae-4ed4-934c-bdf306c13a9e.png">

Reply:
I've updated it and added the last part. What do you think?